packages/core/src/lavadome.mjs

```diff
@@ -41,21 +46,25 @@ export function LavaDome(host, opts) {
     const child = hardened();
     appendChild(shadow, child);
 
-    function text(text) {
-        if (typeof text !== 'string') {
+    let secret = '';
```

Comment:
It doesn't feel great to have the secret hang out in memory in this long-lived scope... Not that I have a proposal on how it could be achieved differently, though...

Reply:
I see your point, but essentially, it's only natural that in order to provide service where the input is somehow manipulated we must have access to it